views/layouts/blank.hbs

```diff
@@ -16,8 +16,14 @@
 </head>
 
 <body>
-  {{{body}}}
 
+  <!-- jquery -->
```

Comment:
Not sure if we want to load all these resources on every blank layout page

Reply:
Understandable